### ISSUE TITLE
Search doesn't work, free account button doesn't load with all cookies disabled

### ISSUE THREAD

**Issue by nr-kkenney** [newrelic/docs-website#2287]
### Related

https://github.com/newrelic/gatsby-theme-newrelic/issues/356

### Issue

It looks like the free account button fails to load, and search fails to work when cookies are disabled. Not sure if those are related or independent -- if search is failing due to the button not loading, or just because of the cookies being disabled.

There might be some discussion about what we might expect when all cookies are disabled, since that is a pretty extreme case for a user to take.

Screenshot:
![Screen Shot 2021-05-13 at 10 22 19 AM](https://user-images.githubusercontent.com/70179215/118162037-254ad800-b3d5-11eb-8ff1-9138c2521806.png)

Steps to repro:
* Disable all cookies
* Navigate to any site (docs, developer, opensource)
* Observe free account button not loading, search not working.

### Acceptance Criteria
- [ ] The Free Account button should appear when a user has disabled all cookies.

**Comment by nr-kkenney** [comment]
aside from search not working & the button not loading, at least one page has its content / organization
 affected by having cookies disabled:
![Screen Shot 2021-05-13 at 3 29 33 PM](https://user-images.githubusercontent.com/70179215/118197179-462a2200-b403-11eb-8376-9ab341d3cb81.png)

**Comment by LizBaker** [comment]
A couple conclusions: the button is not affecting the search, a whoooole lot breaks with disabled cookies and gatsby/ssr. We can at least not have the button be a spinner but the only answer I came up with for that is to get rid of the spinner and allow for a color change to happen on load. 
I did try to read cookie enablement off the window but I can't seem to do anything dynamic with cookies disabled on chrome. Not even a console log comes through so it seems it's all very much stuck in ssr and therefore can't dynamically render a spinner for enabled browsers and a default button for disabled :(
